src/handle-data.js

```diff
@@ -45,12 +45,12 @@ export default async function manageData(model, layout, pageInfo, setPageInfo) {
   const height = Math.min(rowsPerPage, totalVerticalCount - top);
   // When the number of rows is reduced (e.g. confirming selections),
   // you can end up still being on a page that doesn't exist anymore, then go back to the first page and return null
-  if (page > 0 && top >= totalVerticalCount) {
+  if (page > 0 && top >= totalVerticalCount && pageInfo) {
```

Comment:
why `pageInfo` is needed here?

Reply:
This is an infinite scroll signal, that there is no data.  So pagInfo is set to null to avoid the re-renders.

Reply:
I see. Thanks.